src/common/buildConfigEntityAttributes.ts

```diff
@@ -0,0 +1,65 @@
+import { Build, BuildConfiguration } from 'pnc-api-types-ts';
+
+import { TEntityAttributes } from 'common/entityAttributes';
+
+const buildTypeValues: BuildConfiguration['buildType'][] = ['MVN', 'NPM', 'GRADLE', 'SBT'];
+
+interface IExtendedBuildConfig extends BuildConfiguration {
+  buildStatus: Build['status'];
+  actions: any;
+}
+
+export const buildConfigEntityAttributes = {
+  id: {
+    id: 'id',
+    title: 'ID',
+  },
+  name: {
+    id: 'name',
+    title: 'Name',
+    filter: {
+      operator: '=like=',
+      isCustomParam: true,
+    },
+  },
+  description: {
+    id: 'description',
+    title: 'Description',
+  },
+  buildType: {
+    id: 'buildType',
+    title: 'Build Type',
+    values: buildTypeValues,
+    filter: {
+      operator: '==',
+    },
+  },
+  project: {
+    id: 'project',
+    title: 'Project',
+  },
+  creationTime: {
+    id: 'creationTime',
+    title: 'Created',
+  },
+  modificationTime: {
+    id: 'modificationTime',
+    title: 'Modified',
+  },
+  creationUser: {
+    id: 'creationUser',
+    title: 'Created By',
+  },
+  modificationUser: {
+    id: 'modificationUser',
+    title: 'Modified By',
+  },
```

Comment:
these could also be filterable

Reply:
See Jira comments, these are not supported